components/layout/AppHeader.tsx

```diff
@@ -1,10 +1,53 @@
-import { Logout } from "../../components/wallet";
+import { useAccount } from "wagmi";
+import { ConnectWallet, Logout, SIWE } from "../../components/wallet";
+import { useUser } from "hooks";
+import { useCallback } from "react";
+import { Logo } from "components/elements";
+import Link from "next/link";
+
+type AppHeaderProps = {
+  sections?: {name: string}[]
+}
+
+const defaultSections = [
+  {
+    name: 'Home'
+  },
+  {
+    name: 'How it works',
+  },
+  {
+    name: 'Popular Stream',
+  }
+]
+
+export const AppHeader = ({sections = defaultSections}: AppHeaderProps) => {
+  const [{ data: accountData }] = useAccount();
+  const { address: user } = useUser()
+
+  const authButton = useCallback(() => {
+    if (user) {
+      return (
+        <Logout />
+      )
+    } else {
+      return accountData ? <SIWE /> : <ConnectWallet />
+    }
+  }, [user, accountData])
 
-export const AppHeader = () => {
   return (
-    <header className="w-full p-3 flex justify-between items-center fixed top-0 left-0 z-10 bg-white">
-      <h1 className="font-bold text-xl">Glitch App</h1>
-      <Logout />
+    <header className="w-full px-10 py-4 flex justify-between items-center fixed top-0 left-0 z-10">
+        <Link  href={user ? '/dashboard' : '/'}>
+          <Logo />
+        </Link>
+        <div id="sections">
+          <ul className="flex text-gray-100 space-x-8">
+            {sections.map(section => (
+              <li className="hover:text-secondary hover:underline decoration-secondary underline-offset-4 hover:cursor-pointer">{section.name}</li>
+            ))}
```

Comment:
There was a warning here for no key for the child.  I've seen it a couple of ways.  You could do key={section.id}, but then you would have to make sure there was a unique id in the sections object.  Another way that I'm a fan of is using the index of the object like this:

```
{sections.map((section, index) => {
    <li key={index} className="hover:text-secondary hover:underline decoration-secondary underline-offset-4 hover:cursor-pointer">{section.name}</li>
))}
```

This way you get a unique key for every child and don't have to rely on a unique id in the object